content/blog/2020-09-17-delivering-learnr-tutorials-in-a-package/index.Rmarkdown

```diff
@@ -0,0 +1,322 @@
+---
+title: How to deliver learnr tutorials in a package
+author: Desirée De Leon
+date: '2020-09-17'
+slug: delivering-learnr-tutorials-in-a-package
+categories:
+  - package
+  - teach
+tags:
+  - rmarkdown
+  - teach
+  - learnr
+description: |
+  A step by step guide to create a package for a learnr tutorial.
+photo:
+  url: https://unsplash.com/photos/sxNt9g77PE0
+  author: Erda Estremera
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(fig.align = "center", out.width = "50%")
+```
+
+## A package for the learnr occasion
+
+
+Packages are not just for functions and data-- you can create a package to deliver interactive tutorials to an audience, too!
+
+If you want to share your [learnr tutorial](https://rstudio.github.io/learnr/) with a large group of users (e.g. large courses where many students will likely run the same tutorials simultaneously), putting your tutorial inside of a package may actually be one the *best* ways to get your content to your audience. Why? Because after the package has been installed and loaded, users can run your tutorial *locally*-- which means that *you* don't have to worry about potential bandwidth limits and issues that might otherwise arise when learnr tutorials are hosted on external servers.
+
+Putting your learnr tutorial in a package is also free-- and always will be, which means you can also sidestep costs associated with hosting services! Your package doesn't have to be on CRAN, and it doesn't have to contain functions or data (but it can if you want!).
```

Comment:
```suggestion
Putting your learnr tutorial in a package is also free-- and always will be, which means you can also sidestep costs associated with hosting services! Your package doesn't have to be on CRAN and it doesn't have to contain functions or data (but it can if you want!).
```